PacketDefinitions420/PacketServer.cs

```diff
@@ -41,7 +41,20 @@ public void InitServer(ushort port, Dictionary<ulong, string> blowfishKeys, IGam
         {
             _game = game;
             _server = new Host();
-            _server.Create(new Address(_serverHost,port), 32, 32, 0, 0);
+            try
+            {
+                _server.Create(new Address(_serverHost, port), 32, 32, 0, 0);
+            }
+            catch (Exception)
+            {
+                foreach (var process in System.Diagnostics.Process.GetProcessesByName("League of Legends"))
```

Comment:
This is VERY risky. You can kill the wrong process very easily. Please remove this.